pkg/controller/statusmanager/status_manager.go

```diff
@@ -346,7 +347,7 @@ func (status *StatusManager) SetFromPods() {
 	// If all pods are being created, mark deployment as not failing
 	status.SetNotFailing(PodDeployment)
 
-	config := &opv1alpha1.NetworkAddonsConfig{}
+	config := &cnaov1.NetworkAddonsConfig{}
```

Comment:
Don't know what to do with this one, maybe we need a generic interface for This operations that has a v1 implementation...

Reply:
like `etworkAddonsStorageVersion` interface that will just implement stuff like 
`getNetworkAddonsConfig()`
`getSchemeGroupVersion()`
?
I'm liking this more and more by the moment

Reply:
Something like that, and maybe the same to get and set so we can put the  "migration" code there.

Reply:
ok I'll do it in a separate PR.